packages/react/utils/src/index.ts

```diff
@@ -0,0 +1,45 @@
+import { ReadonlySignal, Signal } from "@preact/signals-core";
+import { useSignal } from "@preact/signals-react";
+import { useSignals } from "@preact/signals-react/runtime";
+import { Fragment, createElement, useMemo } from "react";
+
+interface ShowProps<T = boolean> {
+	when: Signal<T> | ReadonlySignal<T>;
+	fallback?: JSX.Element;
```

Comment:
I think we need to switch to importing `JSX`, it's (finally!) no longer a global in React 19 AFAIK. It should exist on the React namespace back to 16.